cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -74,16 +74,16 @@ These functions take multiple values as arguments, and calculate and return an a
 [options="header"]
 |===
 |Function                                       | Description
-|<<functions-avg,avg()>>                        | Returns the average of a numeric column.
-|<<functions-collect,collect()>>                | Returns a list containing all collected values.
-|<<functions-count,count()>>                    | Returns the number of rows.
-|<<functions-max,max()>>                        | Returns the highest value in a numeric column.
-|<<functions-min,min()>>                        | Returns the lowest value in a numeric column.
+|<<functions-avg,avg()>>                        | Returns the average of a set of numeric values.
+|<<functions-collect,collect()>>                | Returns a list containing the values returned by a given expression.
+|<<functions-count,count()>>                    | Returns the number of values or rows.
+|<<functions-max,max()>>                        | Returns the maximum value in a set of values.
+|<<functions-min,min()>>                        | Returns the minimum value in a set of values.
```

Comment:
I think we really need to return to the use of the word column. Or some equivalent way of highlighting that these are not functions, but aggregating expressions.